mesonbuild/interpreter/interpreter.py

```diff
@@ -1651,7 +1651,7 @@ def func_find_library(self, node, args, kwargs):
     @disablerIfNotFound
     @permittedKwargs(permitted_dependency_kwargs)
     @typed_pos_args('dependency', varargs=str, min_varargs=1)
-    def func_dependency(self, node, args, kwargs):
+    def func_dependency(self, node, args, kwargs) -> Dependency:
```

Comment:
Looks like you are missing annotations for the parameters, but overall looks good!

Reply:
That's fine, because that is just the minimum annotations needed in the interpreter to get the wayland module typed. :p